vnext/ReactUWP/Views/ControlViewManager.cpp

```diff
@@ -65,6 +70,20 @@ void ControlViewManager::UpdateProperties(ShadowNodeBase *nodeToUpdate, const fo
   }
 
   Super::UpdateProperties(nodeToUpdate, reactDiffMap);
+
+  if (finalizeBorderRadius) {
+    if (control.try_as<winrt::Windows::UI::Xaml::Controls::IControl7>()) {
+      UpdateCornerRadiusOnElement(nodeToUpdate, control);
+    }
+  }
+}
+
+void ControlViewManager::InitializeDefaultProperties(XamlView view) {
+  // Set the default cornerRadius to 0 for Control: WinUI usually default cornerRadius to 2
```

Comment:
Update comment that customer still see CornerRadius effect on RS5- because WinUI has different implementation and we can't change it